docs/package_directory_structure/README.md

```diff
@@ -0,0 +1,40 @@
+# Package directory structure
+Content entities in Demisto are presented by YAML files.
+
+Automation scripts and integrations [YAML files](https://github.com/demisto/content/blob/master/docs/yaml-file-integration/README.MD) include Javascript/Python code.
+
+Python scripts and integrations are stored in directories as packages.
+
+This is required for running [linting](https://github.com/demisto/content/blob/master/docs/linting/README.md) and [unit testing](https://github.com/demisto/content/blob/master/docs/tests/unit-testing/README.md) on the Python code.
+
+The package is converted into a YAML file, which include all the package components, in Demisto's CI build, using the `package_creator.py` script, and then populated in the Content release.
```

Comment:
one can also use the plug-in with the `Export to Demisto` button

Reply:
yea i wrote that in the bottom
the sentence is here just to clarify how eventually the customer gets a yml out of the package